galaxy_ng/app/api/v3/viewsets/collection.py

```diff
@@ -281,6 +281,18 @@ def move_content(self, request, *args, **kwargs):
         add_task = self._add_content(collection_version, dest_repo)
         remove_task = self._remove_content(collection_version, src_repo)
 
+        if (
+            settings.GALAXY_DEPLOYMENT_MODE == DeploymentMode.INSIGHTS.value and
+            dest_repo == AnsibleRepository.objects.get(name='automation-hub')  # use constant or param
```

Comment:
I've been wondering if there is a good way to have a common "get_the_default_repo()" since it comes up in lots of synclist stuff.